### ISSUE TITLE
Missing dependencies during win64 anaconda installation 

### ISSUE THREAD

**Issue by artzet-s** [openalea/visualea#8]
Hello,

On win64 platform with anaconda installation of visualea from openalea channels :

> `conda create -n visualea python`
> `activate visualea`
> `conda install -c openalea openalea.visualea`

I'had execution error when a launch visualea : 

> `ipython`
> `from openalea.visualea.visualea_script import start_gui`
> ` start_gui()`

Resulting error like : 

>> `ImportError: You must install pyqt-tools`  **due to pyqt package absence**
>> `ImportError: No module named rich_ipython_widget` **due to qtconsole absence**
>> `AttributeError: 'module' object has no attribute 'QTextEdit'` **due to pyqt version 5**

So I fix my problem with : 

`conda install -c openalea pyqt=4.10.4`
`conda install -c openalea qtconsole`

**Comment by pradal** [comment]
Thanks Simon.
I will investigate why pyqt5 is installed. Seems strange.

**Comment by artzet-s** [comment]
No, with just pyqt<5 I have the `ImportError: No module named rich_ipython_widget` you can test if you have the same error.

**Comment by pradal** [comment]
qtconsole is installed with openalea.vpltk. I will update openalea.oalab which install ipython-qtconsole and not qtconsole.
Stay tuned (it takes a long time to rebuild everything on Windows)

**Comment by artzet-s** [comment]
Exactly same bug is still here with the last package uploaded : 
 https://ci.appveyor.com/project/pradal/openalea-recipes

**Comment by pradal** [comment]
Have you removed your cache?

**Comment by artzet-s** [comment]
Yes

**Comment by artzet-s** [comment]
More, Yesterday I removed completely my Miniconda and reinstalled, so I have I pretty clean and stable installation of conda.  And I have removed my cache when a tested again. Proof : 

![3](https://user-images.githubusercontent.com/2472052/38938988-2d699c96-4327-11e8-8f18-48c373e29f02.png)

**Comment by pradal** [comment]
Yes. You download openalea.vpltk 2.0.1 while the right one is 2.0.2
Just try with

conda install -c openalea openalea.visualea openalea.vpltk=2.0.2
If it fails, then we have a bug
